src/txn/txn_ckpt.c

```diff
@@ -2014,6 +2061,11 @@ __checkpoint_lock_dirty_tree(
      */
     WT_ERR(__drop(session, NULL, ckptbase, name, strlen(name)));
```

Comment:
Above this code, there is a loop where we try to drop specific checkpoints, check the following comment:
```
We may be dropping specific checkpoints, check the configuration.
```
We may want to increment the stat there too.